src/main/java/edu/kis/powp/jobs2d/drivers/extensions/ExtensionDriverComposite.java

```diff
@@ -0,0 +1,46 @@
+package edu.kis.powp.jobs2d.drivers.extensions;
+
+import edu.kis.powp.jobs2d.Job2dDriver;
+
+import java.util.*;
+
+public class ExtensionDriverComposite implements ExtensionDriver {
+    private Map<String, ExtensionDriver> extensions = new HashMap<>();
+    private Job2dDriver driver;
```

Comment:
what about a list of "horizontal" extensions (not decorators).

Reply:
I guess that i didnt get what you have meant by "horizontal" extensions. I have a map of extensions which are UsageMonitorExtension and  TransformationDriver (should be called TransformationExtension). The driver describes a type of line to use by extensions. I dont see those "horizontal" extensions, could you explain more?